lightning_pose_app/ui/project.py

```diff
@@ -541,26 +543,62 @@ def _render_streamlit_fn(state: AppState):
         )
         st.write("## To move forward, you will need to complete all the steps in this tab.")
         st.write("##")
-        st.markdown("**Need further help? Check the:**")
-        st.markdown(
-            "App [documentation]"
-            "(https://pose-app.readthedocs.io/en/latest/source/tabs/manage_project.html#)",
-            unsafe_allow_html=True,
-        )
-        st.markdown(
-            "Github [repository](https://github.com/Lightning-Universe/Pose-app.html#)",
-            unsafe_allow_html=True,
-        )
-        st.markdown(
-            "Lightning Pose [documentation]"
-            "(https://lightning-pose.readthedocs.io/en/latest/.html#)",
-            unsafe_allow_html=True,
-        )
-        st.markdown(
-            "Ensemble Kalman Smoother (EKS) [documentation]"
-            "(https://pose-app.readthedocs.io/en/latest/source/tabs/train_infer.html#tab-train-infer-ensemble#)",  # noqa
-            unsafe_allow_html=True,
-        )
+        st.write("**Need further help? Check the documntation**")
+        with st.expander("Link to docs"):
+            st.markdown(
+                "App [documentation]"
+                "(https://pose-app.readthedocs.io/en/latest/source/tabs/manage_project.html#)",
+                unsafe_allow_html=True,
+            )
+            st.markdown(
+                "Lightning Pose [documentation]"
+                "(https://lightning-pose.readthedocs.io/en/latest/.html#)",
+                unsafe_allow_html=True,
+            )
+            st.markdown(
+                "Ensemble Kalman Smoother (EKS) [documentation]"
+                "(https://pose-app.readthedocs.io/en/latest/source/tabs/train_infer.html#tab-train-infer-ensemble#)",  # noqa
+                unsafe_allow_html=True,
+            )
+        st.divider()
+        st.write("# Export project")
+        with st.expander("Expend to see export options"):
+            st_project_name = st.selectbox(
+                "Select a project to export",
+                sorted(state.initialized_projects),
+                help="Currectly there is an option only to export labeled frames and project "
+                     "config file. To export trained models go to the teamspace Drive and "
+                     "download it from there."
+            )
+            st.markdown(
+                "Click here to learn how to [Access your data]"
```

Comment:
I think it's clearer if you remove the "help" bubble and put all the info here - it's a bit confusing if someone sees a button to export their project and under that text that directs them to a link about accessing their data. I'd think "wait aren't I accessing my data by exporting?"

So the markdown could say
```
Project exports only contain frames, labels, and the project config file. To download trained models or the results files from video inference, see [here](https://pose-app.readthedocs.io/en/latest/source/accessing_your_data.html)
```